tests/vmi_configuration_test.go

```diff
@@ -532,22 +532,23 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			})
 
 			It("[test_id:5266]should boot to NIC rom if a boot order was set on a network interface", func() {
-				By("Creating a VMI with no disk and an explicit network interface")
-				vmi := tests.NewRandomVMI()
-				tests.AddExplicitPodNetworkInterface(vmi)
-
-				By("Enabling BIOS serial output")
-				vmi.Spec.Domain.Firmware = &v1.Firmware{
-					Bootloader: &v1.Bootloader{
-						BIOS: &v1.BIOS{
-							UseSerial: tests.NewBool(true),
-						},
-					},
-				}
-
 				By("Enabling network boot")
 				var bootOrder uint = 1
-				vmi.Spec.Domain.Devices.Interfaces[0].BootOrder = &bootOrder
+				interfaceDeviceWithMasqueradeBinding := libvmi.InterfaceDeviceWithMasqueradeBinding()
+				interfaceDeviceWithMasqueradeBinding.BootOrder = &bootOrder
+				resourceMemory := "256Mi"
+				if !checks.IsARM64(testsuite.Arch) {
+					resourceMemory = "128Mi"
+				}
```

Comment:
In a future PR we should remove such things from tests entirely and handle them within renderLaunchManifest.
FYI @maiqueb, it might be related to the PR you're working on